libs/sdk/e2e-schematics/src/generators/component-e2e/index.spec.ts

```diff
@@ -10,8 +10,24 @@ import { Linter } from '@nrwl/linter';
 import componentE2eGenerator from './index';
 
 describe('component-e2e', () => {
-  it('should create e2e infrastructure for a component', async () => {
+  function setupTest() {
     const tree = createTreeWithEmptyWorkspace();
+
+    tree.write(
+      'workspace.json',
```

Comment:
For some reason, Nx doesn't include the `workspace.json` files anymore. Creating them here to get our tests to pass.